config/clusters/qcl/support.values.yaml

```diff
@@ -0,0 +1,35 @@
+prometheusIngressAuthSecret:
+  enabled: true
+
+redirects:
+  rules:
+    - from: staging.qcl.2i2c.cloud
+      to: staging.quantifiedcarbon.com
+
+prometheus:
+  server:
+    ingress:
+      enabled: true
+      hosts:
+        - prometheus.qcl.2i2c.cloud
+      tls:
+        - secretName: prometheus-tls
+          hosts:
+            - prometheus.qcl.2i2c.cloud
+
+grafana:
+  grafana.ini:
+    server:
+      root_url: https://grafana.quantifiedcarbon.com/
+  auth.github:
+    enabled: true
+    allowed_organizations: 2i2c-org
+  ingress:
+    hosts:
+      - grafana.qcl.2i2c.cloud
+      - grafana.quantifiedcarbon.com
+    tls:
+      - secretName: grafana-tls
+        hosts:
+          - grafana.qcl.2i2c.cloud
+          - grafana.quantifiedcarbon.com
```

Comment:
I considered this config quite a bit, thinking that perhaps it was important to list `grafana.quantifiedcarbon.com` as the topmost host. I think it may not be important based on investigation in https://github.com/2i2c-org/infrastructure/issues/2304 though.

What do you think about going for `grafana.quantifiedcarbon.com` as the first entry systematically in lists to reduce a risk of issues for now?

Reply:
I'll read through your investigation on #2304 and this can be updated if need be before we enable the prod hub and handover to the community